lib/tty/prompt/list.rb

```diff
@@ -319,6 +373,30 @@ def render_footer
         colored_footer = @prompt.decorate(@page_help, @help_color)
         "\n" << colored_footer
       end
+
+      # Choices matching the filter, if set; otherwise, all choices
+      #
+      # @return [Choice]
+      #
+      # @api private
+      def visible_choices
```

Comment:
My preference would be to name it simply `choices` as the fact that they are visible is just a presentation concept. Here we're dealing with data transformation and what happens with in the terminal display is of no concern. What follows, I would also make this method public as exposing it may lead to some interesting use cases where people can inspect their current choices.

Reply:
There is a already a [:choices method](https://github.com/saveriomiroddi/tty-prompt-dev/blob/a16277fb21561502bc5ab18ce7ba16eb80bd1567/lib/tty/prompt/list.rb#L160) :smile:

Reply:
Hmmm...with few methods I apply a style, in which the `choices` could act as reader and writer of the values. When no value is provided it returns the filtered/all choices:

```ruby
def choices(values = (not_set = true))
  if not_set
     filtered/all choices....
  else
     current behaviour....
  end
end
```
How do you feel about it?

Reply:
He! I've never seen it, so it seems strange/unusual. It's a trivial change though, so I've just applied it.

Let me know if you think unit testing is required, since in `select_spec.rb` there is none (at least, based on what I see :smile:), but both getter and setter version are tested extensively (functionally).